README.md

```diff
@@ -55,6 +56,31 @@ You also may need to change the source URIs for components. We recognize this is
 
 If you have your own mirror of the components of puppet-agent, you can also use a rewrite rule. See the [Vanagon README](https://github.com/puppetlabs/vanagon/blob/master/examples/projects/project.rb#L26) for an example.
 
+
+Building puppet-agent for windows
```

Comment:
FYI it looks like this is reverting 424c8c6, which I think we want to keep eventually (updating readme to not reference puppet_for_the_win) - or does this revert mean we go back to using puppet_for_the_win, not vanagon for windows builds? I assume not. (this comment doesn't block merge)